execution-engine/Cargo.toml

```diff
@@ -24,6 +24,8 @@ std = [
 ]
 
 [dependencies]
+use  log :: info
```

Comment:
It should be in the syntax of `log = "0.4.13"`, meaning, import log crate (or package) of version 0.4.13